app/views/proposal_fields/_text.html.erb

```diff
@@ -1,6 +1,6 @@
 <div data-controller="proposal-form" data-proposal-form-propid="<%= @proposal&.id %>">
 <div>
-  <label for="statement"><%= field.statement %></label>
+  <label for="statement"><%= field.statement %></label><%= mendatory_field?(field) ? print_validation : '' %>
```

Comment:
Should move the conditional to the helper method, instead of using it in every view partial.